packages/pocket-ic/src/lib.rs

```diff
@@ -1464,7 +1464,7 @@ mod test {
         );
         for ic_error_code in ic_error_types::ErrorCode::iter() {
             let error_code: ErrorCode = (ic_error_code as u64).try_into().unwrap();
-            assert_eq!(ic_error_code as u64, error_code as u64);
```

Comment:
Equality of `u64`s follow from the test `error_code_round_trip` and thus this commit adds equality assertion for the error code debug prints. Does this look fine to you, @michael-weigelt ?

Reply:
I see no problem with this